addons/event/models/event_ticket.py

```diff
@@ -136,10 +136,13 @@ def _compute_seats(self):
             if ticket.seats_max > 0:
                 ticket.seats_available = ticket.seats_max - (ticket.seats_reserved + ticket.seats_used)
 
-    @api.depends('seats_limited', 'seats_available')
+    @api.depends('seats_limited', 'seats_available', 'event_id.event_registrations_sold_out')
     def _compute_is_sold_out(self):
         for ticket in self:
-            ticket.is_sold_out = ticket.seats_limited and not ticket.seats_available
+            ticket.is_sold_out = (
```

Comment:
Great work 🙂,  just curious why don't just:

```suggestion
            ticket.is_sold_out = ticket.seats_limited and not ticket.seats_available) or ticket.event_id.event_registrations_sold_out
```

Reply:
RD-Sales styleguide habit, but yeah, I guess there's no char limit on lines in this scope 😄 

The benefit is that if the conditional requires change in a future commit, you can add an extra clause without replacing the entire line, making diffs easier to read while maintaining a more granular git-blame history.